waifus/rin hoshizora.json

```diff
@@ -0,0 +1,14 @@
+{
+	"type" : 0,
+	"japanese-name" : "星空凛",
+	"english-name" : "Rin Hoshizora",
+	"month" : 11,
+	"day-of-month" : 1,
+	"age-group-by-appearance" : "teenager",
+	"definitely-legal": false,
+	"age-in-show": 15,
+	"is-a-trap": false,
+	"image" : "https://www.clipartwiki.com/clipimg/full/126-1264823_hoshizora-rin-png-rin-hoshizora-render.png",
+	"notes": ["Rinboi."],
```

Comment:
I'm not sure what this means. Can you explain? Everything looks good tho.

Reply:
'haha rinboi' is like a joke in the fanbase cuz she looks masculine